src/corpus/domain/chapter.ts

```diff
@@ -142,7 +143,9 @@ export class ChapterDisplay {
   }
 
   get url(): string {
-    return `https://www.ebl.lmu.de/corpus/${this.idParts
+    const idParts: [string, number, number, string, string] = [...this.idParts]
```

Comment:
brauchst hier nicht den type extra dazu schreiben da ja schon `this.idParts ` den return type hat